recipe/meta.yaml

```diff
@@ -1,38 +1,37 @@
-    sha256: b3576ffa2a70e7704bdf7febe7109f4cfb416a0178222b58f5ee0fd455980337  # [aarch64 or arm64]
+  - url: https://github.com/ARM-software/ComputeLibrary/archive/refs/tags/v24.09.tar.gz  # [aarch64 or arm64]
+    sha256: 49b8620f21cbbe49e825a131d9eacd548532646289b50e070b83860bd88087fe  # [aarch64 or arm64]
     folder: src/plugins/intel_cpu/thirdparty/ComputeLibrary  # [aarch64 or arm64]
-  - url: https://github.com/openvinotoolkit/oneDNN/archive/c8ae8d96e963bd04214858319fa334968e5e73c9.tar.gz
-    sha256: b9c2a53061b4528231ff5fbcee85900d698c329c7977b1f39c5d3d65f29c2caa
+  - url: https://github.com/openvinotoolkit/oneDNN/archive/c60a9946aa2386890e5c9f5587974facb7624227.tar.gz
+    sha256: 37cea8af9772053fd6d178817f64d59e3aa7de9fd8f1aa21873075bb0664240f
     folder: src/plugins/intel_cpu/thirdparty/onednn
   - url: https://github.com/openvinotoolkit/mlas/archive/d1bc25ec4660cddd87804fcf03b2411b5dfb2e94.tar.gz
     sha256: 0a44fbfd4b13e8609d66ddac4b11a27c90c1074cde5244c91ad197901666004c
     folder: src/plugins/intel_cpu/thirdparty/mlas
-  - url: https://github.com/oneapi-src/oneDNN/archive/refs/tags/v3.6-pc.tar.gz  # [win64 or linux64]
-    sha256: d0eeff1425e318887c429a709d58010a580c8173eba8fbee74bca41b7f621f59  # [win64 or linux64]
+  - url: https://github.com/oneapi-src/oneDNN/archive/1722066ad4c0f15495f2d0fcbe9deb2bfd188c36.tar.gz  # [win64 or linux64]
+    sha256: 8955f19800066d8a705df2eb8b462085d6142b4f771ff88ae196e59a0d4ab1bb  # [win64 or linux64]
     folder: src/plugins/intel_gpu/thirdparty/onednn_gpu  # [win64 or linux64]
-  - url: https://github.com/oneapi-src/level-zero/archive/refs/tags/v1.17.6.tar.gz  # [linux64]
-    sha256: a018923fa0732c4fb9228ff668124363de18f376a314113aa6aaf0a6b5d1ab3b  # [linux64]
-    folder: src/plugins/intel_npu/thirdparty/level-zero  # [linux64]
-  - url: https://github.com/intel/level-zero-npu-extensions/archive/16c85231a82ee1a0b06ed7ab7da3f411a0878ed7.tar.gz  # [linux64]
-    sha256: cde001be74324d8370d4dcdb8457afe5a6a1b3a958b05190819ffe62f25c2971  # [linux64]
+  - url: https://github.com/intel/level-zero-npu-extensions/archive/a6487cc2c5da9aa13db9e005a320a1b6a0ee5919.tar.gz  # [linux64]
+    sha256: 1e7b4140eed15aec2be36075a322e352013292b28e842608476e9d1ce3b77959  # [linux64]
     folder: src/plugins/intel_npu/thirdparty/level-zero-ext  # [linux64]
-  - url: https://github.com/onnx/onnx/archive/refs/tags/v1.16.0.tar.gz
-    sha256: 0ce153e26ce2c00afca01c331a447d86fbf21b166b640551fe04258b4acfc6a4
+  - url: https://github.com/onnx/onnx/archive/refs/tags/v1.17.0.tar.gz
+    sha256: 8d5e983c36037003615e5a02d36b18fc286541bf52de1a78f6cf9f32005a820e
     folder: thirdparty/onnx/onnx
+  - url: https://github.com/oneapi-src/level-zero/archive/fd459f28041c176df23a0b7b791ff20a7689d237.tar.gz  # [linux64]
+    sha256: e078cae698e368c4e8e57d268f4ef0b766ed8f5548ac8e4fc0a07f4c406bb50d  # [linux64]
+    folder: thirdparty/level-zero/level-zero  # [linux64]
```

Comment:
I’m not too sure what level-zero is but can we try to use the conda-forge package?

https://github.com/conda-forge/level-zero-feedstock

Reply:
Currently - not.

- OpenVINO NPU plugin depends on older version of level-zero. PR to upgrade https://github.com/openvinotoolkit/openvino/pull/27632
- level-zero's pkg-config files are broken and fix is https://github.com/oneapi-src/level-zero/pull/243